app/forms/idv/api_image_upload_form.rb

```diff
@@ -197,20 +197,21 @@ def update_analytics(client_response)
       update_funnel(client_response)
       track_event(
         Analytics::IDV_DOC_AUTH_SUBMITTED_IMAGE_UPLOAD_VENDOR,
-        client_response.to_h.merge(image_metadata),
+        client_response.to_h.merge(client_image_metrics: image_metadata),
       )
     end
 
     def image_metadata
       params.permit(:front_image_metadata, :back_image_metadata).
         to_h.
         transform_values do |str|
-          JSON.parse(str, symbolize_names: true)
+          JSON.parse(str)
         rescue JSON::ParserError
           nil
         end.
         compact.
-        symbolize_keys
+        transform_keys { |key| key.gsub(/_image_metadata$/, '') }.
+        deep_symbolize_keys
```

Comment:
I'd have preferred to have kept `symbolize_names`, and maybe even apply `.to_sym` as part of `transform_keys`, but for some reason the nested values of the parsed JSON would change from symbols back to strings as a result of the call to `transform_keys` 🤷

Reply:
Huh, strange. That's not something I have run into. I think the scope is limited enough that this implementation is fine.